AdvanceTaskMarsPart1/Pages/NotificationPage.cs

```diff
@@ -0,0 +1,105 @@
+﻿using AdvanceTaskMarsPart1.Utilities;
+using OpenQA.Selenium;
+
+namespace AdvanceTaskMarsPart1.Pages
+{
+    public class NotificationPage : CommonDriver
+    {
+        private IWebElement NotificationTab   => driver.FindElement(By.XPath("//div[@class='ui top left pointing dropdown item']"));
+        private IWebElement SeeAllLink        => driver.FindElement(By.XPath("//a[text()='See All...']"));
+        private IWebElement LoadMoreButton    => driver.FindElement(By.XPath("//a[@class='ui button']"));
+        private IReadOnlyCollection<IWebElement> RequestsList => driver.FindElements(By.XPath("//div[@class='fourteen wide column']"));
+        private IWebElement ShowLessButton    => driver.FindElement(By.XPath("//a[text()='...Show Less']"));
+        private IWebElement SelectCheckbox    => driver.FindElement(By.XPath("(//div[@class='one wide column']/input[@type='checkbox'])[1]"));
+        private IWebElement DeleteButton      => driver.FindElement(By.XPath("//div[@data-tooltip='Delete selection']"));
+        private IWebElement Message           => driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        private IWebElement MarkSelectionAsRead => driver.FindElement(By.XPath("//i[@class='check square icon']"));
+        private IWebElement SelectAllButton     => driver.FindElement(By.XPath("//div[@data-tooltip='Select all']"));
+        private IWebElement UnSelectAllButton   => driver.FindElement(By.XPath("//i[@class='ban icon']"));
+        private IReadOnlyCollection<IWebElement> CheckBoxSelected => driver.FindElements(By.XPath("//input[@type='checkbox']"));
+
+        public void LoadMore_Notification()
+        {
+            Wait.WaitToBeClickable("XPath", "//div[@class='ui top left pointing dropdown item']", 4);
+            NotificationTab.Click();
+            Wait.WaitToBeClickable("XPath", "//a[text()='See All...']", 4);
+            SeeAllLink.Click();
```

Comment:
So you have not created a framework which breaks down the components down! This is mentioned clearly in the documentation and there is also an example of how this should be done! Please update the framework so that it is more readable and maintainable